inspirehep/modules/authors/templates/authors/forms/update_form.html

```diff
@@ -52,9 +53,9 @@
 
     var config = {
       form: {
-        save_url: '{{ url_for("inspire_authors.validate") }}',
-        save_all_url: '{{ url_for("inspire_authors.validate", all='1') }}',
-        complete_url: '{{ url_for("inspire_authors.validate", submit='1') }}',
+        save_url: '{{ url_for("inspire_authors.validate", is_update=is_update) }}',
+        save_all_url: '{{ url_for("inspire_authors.validate", all='1', is_update=is_update) }}',
+        complete_url: '{{ url_for("inspire_authors.validate", submit='1', is_update=is_update) }}',
```

Comment:
Does that means that the `is_update` flag can in principle be tweaked client side? Can't you always know if it's an update just by looking server-side?